src/App/Pages/Accounts/LoginPasswordlessViewModel.cs

```diff
@@ -92,6 +93,15 @@ private FormattedString CreateFingerprintPhrase(string fingerprintPhrase)
             }
         }
 
+        private async Task PasswordlessLoginAsync(bool approveRequest)
+        {
+            await _deviceActionService.ShowLoadingAsync(AppResources.Loading);
+            var res = await _authService.PasswordlessLoginAsync(LoginRequest.Id, LoginRequest.PubKey, approveRequest);
```

Comment:
Do you need the `res` for something? If you don't, delete the var assignment.
I see that the `PasswordlessLoginResponse` has a `RequestApproved` property, shouldn't that be checked to show the toast to see whether the request has been approved or not? (I don't know if the user can approve it on mobile and server can deny it therefore even though the user approved it the toast should show "Denied" or some extra handling should be made like not popping the modal)